frontend/src/locales/zh-CN.yml

```diff
@@ -184,20 +186,20 @@ activity:
   created: 创建于
   invoker: 执行者
   sentence:
-    created-issue: "创建工单"
```

Comment:
This diff is still annoying, on my local environment, it changes double quote to single quote

@xiaoluoboding @LiuJi-Jim @ecmadao any idea?

Reply:
This PR is only for i18n.

I will do some research on how to lint `YAML/YML` files.

PTAL